utils/build_swift/defaults.py

```diff
@@ -40,7 +40,7 @@
 CMAKE_GENERATOR = 'Ninja'
 
 COMPILER_VENDOR = 'none'
-SWIFT_USER_VISIBLE_VERSION = CompilerVersion('4.1')
+SWIFT_USER_VISIBLE_VERSION = CompilerVersion('4.2')
 CLANG_USER_VISIBLE_VERSION = CompilerVersion('5.0.0')
```

Comment:
This default is very suspicious. Shouldn't we be getting it from the Clang repository somehow? cc @Rostepher